tests/config/test_character_configuration.py

```diff
@@ -0,0 +1,150 @@
+import json
+import os
+import tempfile
+
+import pytest
+from constant_sorrow.constants import NO_KEYRING_ATTACHED, CERTIFICATE_NOT_SAVED
+
+from nucypher.characters.chaotic import Felix
+from nucypher.characters.lawful import Alice, Bob
+from nucypher.characters.lawful import Ursula
+from nucypher.config.characters import AliceConfiguration, BobConfiguration, FelixConfiguration
+from nucypher.config.characters import UrsulaConfiguration
+from nucypher.config.storages import ForgetfulNodeStorage
+from nucypher.utilities.sandbox.constants import TEMPORARY_DOMAIN
+
+# Main Cast
+configurations = (AliceConfiguration, BobConfiguration, UrsulaConfiguration)
+characters = (Alice, Bob, Ursula)
+
+# Auxiliary Support
+blockchain_only_configurations = (FelixConfiguration, )
+blockchain_only_characters = (Felix, )
+
+# Assemble
+characters_and_configurations = list(zip(characters, configurations))
+all_characters = tuple(list(characters) + list(blockchain_only_characters))
```

Comment:
I think, you can sum up tuples without casting to `list`

Reply:
:heavy_check_mark:  - Fixed in 8e9e08a0cf4bda13e7a4428e846da5dacb7bbe14